jOOQ-examples/jOOQ-spring-boot-example/pom.xml

```diff
@@ -39,7 +39,6 @@
         <dependency>
             <groupId>org.jooq</groupId>
             <artifactId>jooq</artifactId>
-            <version>${org.jooq.version}</version>
```

Comment:
Same here. We could reference some external or default jOOQ version instead of the latest snapshot build from here on GitHub, but that would incur quite some maintenance work to keep up to date...